Fst2List.cpp

```diff
@@ -1979,18 +1983,39 @@ int CFstApp::outWordsOfGraph(int depth) {
   return 0;
 }
 
-//
 //
 //
 //
 
-// TODO(jhondoe) Use UnitexGetOpt instead argc, argv
+const char* optstring_Fst2List=":o:Sp:a:t:l:i:mdf:vVhs:r:c:";
+const struct option_TS lopts_Fst2List[]= {
+  {"output",required_argument_TS,NULL,'o'},
+  {"automaton",required_argument_TS,NULL,'a'},
+  {"transducer",required_argument_TS,NULL,'t'},
+  {"limit",required_argument_TS,NULL,'l'},
+  {"stop-graph",required_argument_TS,NULL,'i'},
+  {"print-mode",required_argument_TS,NULL,'p'},
+  {"mode",no_argument_TS,NULL,'m'},
+  {"debug",no_argument_TS,NULL,'d'},
```

Comment:
Maybe I'm missing something here. Why `d`, formerly `-d : disable loop check` has now 'debug' as long flag ?

Reply:
My bad. I got mixed up.
I wanted to suggest making `-d` (disable loop check) the default behaviour. As of now I don't see the benefit in having that option as there is no way to use it in the IDE.
In my opinion it'd make more sense to have a `-d, --debug` or `--enable_loop_check` option to get the extra info about loops.

Reply:
As I told you before, it would not be possible to take out or change the semantic of a former option without deprecate it and offer a backward compatibility. BTW, Unitex-based tools are commonly deployed without the IDE, so there is no direct relationship between the importance of an option and its availability through the IDE. I hope it's clearer now.